### PR TITLE
Dutch Locales

### PR DESCRIPTION
**Pull request by ChatDisabled**

### CODE REVIEW

**Comment by itschip** [comment]
Please add this translation to the config. [Reference](https://github.com/project-error/npwd/pull/414/files)

**Comment by ChatDisabled** [comment]
🤸‍♂️

File: phone/src/locale/nl.json [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+      "PROFILE_CONFIGURATION": "App is onjuist geconfigureerd - kon profiel niet maken. Gelieve staff te contacteren.",
+      "EDIT_PROFILE_IMAGE": "Foto URL",
+      "EDIT_PROFILE_NAME": "Profiel naam",
+      "EDIT_PROFILE_BIO": "Bio",
+      "EDIT_PROFILE_LOCATION": "Locatie",
+      "EDIT_PROFILE_JOB": "Baan",
+      "EDIT_PROFILE_TAGS": "Tags",
+      "EDIT_PROFILE_PREVIEW": "Preview Bekijken",
+      "LIKE": "Like",
+      "DISLIKE": "Dislike"
```
Comment (itschip) [inline phone/src/locale/nl.json:160]: Are these correct?

File: phone/src/locale/nl.json [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    "GENERIC_UPDATE": "Update",
+    "GENERIC_REQUIRED": "Benodigd",
+    "GENERIC_DELETE": "Verwijder",
+    "GENERIC_REPORT": "Rapporteer",
+    "GENERIC_CALL": "Bel",
+    "GENERIC_MESSAGE": "Bericht",
+    "GENERIC_CLOSE": "Sluit",
+    "GENERIC_CANCEL": "Terug",
+    "GENERIC_TITLE": "Titel",
+    "GENERIC_CONTENT": "Content",
```
Comment (itschip) [inline phone/src/locale/nl.json:278]: Is this correct?

File: phone/src/locale/nl.json [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+      },
+      "MESSAGES": {
+        "NEW_RETWEET_BROADCAST": "@{{ profile_name }} retweeted iemand!",
+        "NEW_BROADCAST": "Nieuwe tweet van @{{ profile_name }}!",
+        "UNREAD_MESSAGES": "Jij hebt {{ count }} nieuwe tweets in jouw tijdlijn!"
+      },
+      "SUBMIT_IMAGE": "Media Toevoegen",
+      "TWEET": "Tweet",
+      "REPORTED": " Reported",
+      "REPORT": "Report",
```
Comment (itschip) [inline phone/src/locale/nl.json:115]: Are these correct?

**Comment by ChatDisabled** [comment]
To respond to them all, Dutch copies a lot of words straight out of English since they are similar anyway or we can't create a new word for it.
[Source](https://www.vlaanderen.be/taaladvies/geliket-geliked)

But the last two could be properly translated yeah

**Comment by itschip** [comment]
> To respond to them all, Dutch copies a lot of words straight out of English since they are similar anyway or we can't create a new word for it. [Source](https://www.vlaanderen.be/taaladvies/geliket-geliked)
> 
> But the last two could be properly translated yeah

Fine, if you're able to resolve the conflicts, I think we can merge this.

**Comment by TasoOneAsia** [comment]
@itschip CI failure though?

**Comment by itschip** [comment]
> CI failure though?
@TasoOneAsia Already fixed in dev

**Pull request merged**